README.md

```diff
@@ -9,10 +9,46 @@ clone the repository.
 
 # Usage
 In order to compile a function you have to supply it as a string to the `transpile` function.
-The returned function takes the name of the function that you want to call as the first parameter.
-Any further parameters get passed to the WebAssembly instance.
+The returned value is of the type CallWrapper.
 
 ```javascript
-const exports = Transpiler.transpile('function add(a, b) { return a + b; }');
-const result = exports('add', 12, 30);
+const wrapper = new Transpiler().transpile('function add(a, b) { return a + b; }');
+wrapper.setFunctionName('add').call(21, 21);
 ```
+
+# API
+## Transpiler creation
```

Comment:
Is it necessary to distinguish between the creation and the manipulation of the transpiler?